bk-config/bin/create_registration_cert.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+
+
+if [ $# -eq 0 ]
```

Comment:
Obviously this doesn't allow for a registration key to be created *without* an expiration time.  I am totally cool with this.  Just checking that was your intention.

Reply:
I do not think we would want to support "without expiration", but I could imagine we want to support "months" or "weeks". That feature we should add soon.